src/main/scala/ai/privado/languageEngine/javascript/tagger/sink/JSAPITagger.scala

```diff
@@ -52,6 +52,16 @@ import overflowdb.BatchedUpdate
 class JSAPITagger(cpg: Cpg, ruleCache: RuleCache, privadoInput: PrivadoInput)
     extends APITagger(cpg, ruleCache, privadoInput) {
 
+  override val apis = cacheCall
+    .name(APISINKS_REGEX)
+    .methodFullNameNot(COMMON_IGNORED_SINKS_REGEX)
+    .code(commonHttpPackages)
+    .l ++ cacheCall
+    .name(APISINKS_REGEX)
+    .methodFullNameNot(COMMON_IGNORED_SINKS_REGEX)
+    .methodFullName(commonHttpPackages)
```

Comment:
@pandurangpatil Approving this for now, an elegant and optimized way to write this would be 
```
    cacheCall
    .name(APISINKS_REGEX)
    .methodFullNameNot(COMMON_IGNORED_SINKS_REGEX)
    .or(_.code(commonHttpPackages), _.methodFullName(commonHttpPackages)).l

Reply:
already done that